x-pack/plugins/ingest_manager/common/types/rest_spec/epm.ts

```diff
@@ -71,6 +71,26 @@ export interface InstallPackageResponse {
   response: AssetReference[];
 }
 
+export interface UpgradePackageError {
```

Comment:
This name reads like it's an actual Error. Mind prefixing with an `I` for `IUpgradePackageError` or adding something to the end?